### ISSUE TITLE
tbl_regression does not support glm when family = gaussian

### ISSUE THREAD

**Issue by michaelcurry1123** [ddsjoberg/gtsummary#62]
```
mod1 <-   glm(vs ~ cyl,family = binomial, data =mtcars ) %>% 
          tbl_regression()


mod2 <-   glm(mpg ~ cyl,family = gaussian, data =mtcars ) %>% 
  tbl_regression()

```
first model runs fine second does not

**Comment by ddsjoberg** [comment]
Bug fixed in commit d5c6b5604382476229405415b94a97f93967d203

**Comment by ddsjoberg** [comment]
THANK YOU SO MUCH FOR THE CHECKS!!  MUCH MUCH APPRECIATED! :)

**Comment by ddsjoberg** [comment]
Fix is in PR #65
